touchscreen.lua

```diff
@@ -156,7 +156,10 @@ local function create_formspec(meta, data)
 	if meta:get_int("no_prepend") == 1 then
 		fs = fs.."no_prepend[]"
 	end
-	if not meta:get("real_coordinates") and not meta:get("realcoordinates") then
+	if (
+		meta:contains("real_coordinates") and meta:get_int("real_coordinates")
+		or meta:get_int("realcoordinates") -- old screen
+	) == 0 then
```

Comment:
This should do the same thing with less code:
```suggestion
	if (meta:get("real_coordinates") or meta:get("realcoordinates")) ~= "1" then
```

Reply:
could make the argument for dropping/depreciating any new non real coords since only would lose 5.0

Reply:
changed it... I see how it works, but I'm not sure whether I like it to call `set_int` in one place, but check for the string value later...